modules/internal/NetworkAtomics.chpl

```diff
@@ -32,6 +32,17 @@ module NetworkAtomics {
   record RAtomicBool {
     var _v: int(64);
 
+    pragma "no doc"
+    proc init=(other : RAtomicBool) {
+      this._v = other._v;
+    }
+
+    pragma "no doc"
+    proc init=(other : bool) {
+      this.complete();
+      this.write(other);
```

Comment:
Any real to not just use `this._v = other:int(64);`? Atomic init isn't atomic, so I don't think we need to use `this.write here`